### ISSUE TITLE
error AccessibilityInfo

### ISSUE THREAD

**Issue by KH93b** [hayanisaid/rnal#25]
Unable to resolve "AccessibilityInfo" from "node_modules\rnal\node_modules\react-native\Libraries\react-native\react-native-implementation.js"

**Comment by kamilcessak** [comment]
still taking place

**Comment by ddtch** [comment]
Same here
